dist/images/kubectl-ko

```diff
@@ -949,159 +941,107 @@ env-check(){
+    nodeName=$(kubectl get pod "$pod" -n kube-system -o jsonpath={.spec.nodeName})
+    mkdir -p ./kubectl-ko-log/$nodeName/$component_param
     if [[ "$sub_component_param" == "dmesg" ]]; then
-      kubectl exec $pod -n kube-system -- dmesg -T $( [ "$tail_lines_param" == "all" ] || echo "| tail -n $tail_lines_param" ) || :
+      kubectl exec $pod -n kube-system -- dmesg -T > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
     elif [[ "$sub_component_param" == "iptables" ]]; then
-      echo "******************legacy filter ************************"
-      kubectl exec $pod -n kube-system -- iptables -S || :
-      echo "*********************nft filter ************************"
-      kubectl exec $pod -n kube-system -- /usr/sbin/iptables-nft -S || :
+      echo "******************legacy filter ************************" > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param
+      kubectl exec $pod -n kube-system -- /usr/sbin/iptables-legacy -S >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
+      echo "*********************nft filter ************************" >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param
+      kubectl exec $pod -n kube-system -- /usr/sbin/iptables-nft -S >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param 2>/dev/null || :
 
-      echo "****************** legacy nat ************************"
-      kubectl exec $pod -n kube-system -- iptables -S -t nat || :
-      echo "********************* nft nat ************************"
-      kubectl exec $pod -n kube-system -- /usr/sbin/iptables-nft -S -t nat || :
+      echo "****************** legacy nat ************************" >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param
+      kubectl exec $pod -n kube-system -- /usr/sbin/iptables-legacy -S -t nat >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
+      echo "********************* nft nat ************************" >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param
+      kubectl exec $pod -n kube-system -- /usr/sbin/iptables-nft -S -t nat >> ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param 2>/dev/null || :
 
     elif [[ "$sub_component_param" == "route" ]]; then
-      kubectl exec $pod -n kube-system -- ip route show || :
+      kubectl exec $pod -n kube-system -- ip route show > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
     elif [[ "$sub_component_param" == "link" ]]; then
-      kubectl exec $pod -n kube-system -- ip -d link show || :
+      kubectl exec $pod -n kube-system -- ip -d link show > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
     elif [[ "$sub_component_param" == "neigh" ]]; then
-      kubectl exec $pod -n kube-system -- ip n || :
+      kubectl exec $pod -n kube-system -- ip n > ./kubectl-ko-log/$nodeName/$component_param/$sub_component_param || :
```

Comment:
1. We should also collect IPv6 routes/neighbors;
2. We should also collect IP addresses.

Reply:
added